add-on/panels/js/roomStore.js

```diff
@@ -124,6 +125,13 @@ loop.store = loop.store || {};
     },
 
     /**
+     * Pass a sharedAction.SetOwnDisplayName from chrome to content space.
+     */
+    setOwnDisplayName: function(actionData) {
+      loop.request("SetOwnDisplayName", actionData);
```

Comment:
That makes sense. This patch has no changes to `panel.jsx` the action dispatch was already happening there so I tried to follow that pattern.